nomad/operator_endpoint.go

```diff
@@ -124,3 +125,161 @@ REMOVE:
 	op.srv.logger.Printf("[WARN] nomad.operator: Removed Raft peer %q", args.Address)
 	return nil
 }
+
+// RaftRemovePeerByID is used to kick a stale peer (one that is in the Raft
+// quorum but no longer known to Serf or the catalog) by address in the form of
+// "IP:port". The reply argument is not used, but is required to fulfill the RPC
+// interface.
+func (op *Operator) RaftRemovePeerByID(args *structs.RaftPeerByIDRequest, reply *struct{}) error {
+	if done, err := op.srv.forward("Operator.RaftRemovePeerByID", args, args, reply); done {
+		return err
+	}
+
+	// Check management permissions
+	if aclObj, err := op.srv.ResolveToken(args.AuthToken); err != nil {
+		return err
+	} else if aclObj != nil && !aclObj.IsManagement() {
```

Comment:
Why not OperatorWrite?

Reply:
That seems fine to me; the rest of the raft methods in here take a management token though